app/views/partials/DependenciesByEnvironmentPartial.scala.html

```diff
@@ -17,110 +17,79 @@
 @import partials.ServiceDependenciesPartial
 @import uk.gov.hmrc.cataloguefrontend.connector.{DeploymentVO, RepositoryDetails, TargetEnvironment}
 @import uk.gov.hmrc.cataloguefrontend.connector.model.{Dependencies, Dependency, Version}
-@import uk.gov.hmrc.cataloguefrontend.model.Environment
-@import uk.gov.hmrc.cataloguefrontend.shuttering.ShutterState
-@import uk.gov.hmrc.cataloguefrontend.{ServiceInfoView, ViewMessages}
+@import uk.gov.hmrc.cataloguefrontend.model.{Environment, SlugInfoFlag}
+@import uk.gov.hmrc.cataloguefrontend.shuttering.{ShutterState, ShutterStatusValue}
+@import uk.gov.hmrc.cataloguefrontend.{EnvData, ServiceInfoView, ViewMessages}
 
 
 @this(dependenciesPartial: ServiceDependenciesPartial, viewMessages: ViewMessages)
 
-@(service                      : RepositoryDetails,
-  latestDependencies           : Seq[Dependency],
-  dependenciesByEnvironmentName: Map[Environment, Seq[Dependency]],
-  versionByEnvironmentName     : Map[Environment, Version],
-  latestVersion                : Option[Version],
-  shutterState                 : Map[Environment, ShutterState]
+@(envDatas: Map[SlugInfoFlag, Option[EnvData]]
 )(implicit messages: Messages)
 
 <ul id="environmentTabs" class="nav nav-tabs" role="tablist">
-    <!-- we hard-code 'latest' as it is not a deployment 'environment' -->
-    <li class="nav-item active">
-        <a id="latest-tab" class="nav-link" data-toggle="tab" href="#latest-pane" role="tab" aria-controls="latest" aria-selected="true">
+  @for((slugInfoFlag, optEnvData) <- envDatas.toSeq.sortBy(_._1)) {
```

Comment:
I've forgotten what _1 is here by now. Can we sort elsewhere, or add type info to it?